### PR TITLE
ORION-50 Add servlet for serving configuration preferences

### PR DESCRIPTION
**Pull request by aaudiber**

### CODE REVIEW

**Comment by aaudiber** [comment]
@mingyukim

**Comment by aaudiber** [comment]
We only serve configuration that has a key starting with "plugins.". This is checked on line 59.

**Comment by w10d** [comment]
Ok, seems fine then.

File: bundles/org.eclipse.orion.server.servlets/src/org/eclipse/orion/internal/server/servlets/plugin/ConfigurationServlet.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+/**
+ * Servlet that returns server configuration.
+ *
+ * For security reasons, only configuration keys starting with "plugin." will be exposed by
+ * this servlet.
+ * TODO: Add a whitelist of non-plugin-prefixed configuration keys which may be accessed.
+ */
+public class ConfigurationServlet extends OrionServlet {
+
+	private static final long serialVersionUID = -3989265174924783964L;
```
Comment (mingyukim) [inline bundles/org.eclipse.orion.server.servlets/src/org/eclipse/orion/internal/server/servlets/plugin/ConfigurationServlet.java]: Is it their convention to generate a random Uid?

**Comment by aaudiber** [comment]
@mingyukim They go back and forth between default and generated serial version UIDs. I've switched to default.

**Comment by aaudiber** [comment]
@mingyukim Are you good with merging this now?

**Comment by mingyukim** [comment]
+1

**Pull request merged**